dataproc/create_cluster_json.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from google.cloud import dataproc_v1beta2
+import google.protobuf.json_format as json_format
+import google.cloud.dataproc_v1beta2.proto.clusters_pb2 as clusters
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "D:\\GCP\\xxxx.json"
+
+def create_cluster(project_id,region):
+    '''
+    This function takes an external JSON with cluster configuration and creates the cluster.
+    Currently, there are no examples available to do this and I learnt it the hard way. The
+    function takes the JSON message and converts to protocol buffer message format
+    :param project_id: project id of the service account
+    :param region: region where needs to be created
+    :return: returns the cluster client instance
+    '''
+    cluster_client = dataproc_v1beta2.ClusterControllerClient(client_options={
+        'api_endpoint':'{}-dataproc.googleapis.com:443'.format(region)
+    })
+
+    ##The JOSN is created under the clusterconfig folder
+    with open("./clusterconfig/dataproc-cluster.json") as f:
+        cluster = f.read()
+        cluster_message = json_format.Parse(cluster,clusters.Cluster())
```

Comment:
These aren't entirely explicit what these are representing just by their variable names. Perhaps `cluster_json` and `cluster_config`, for `cluster` and `cluster_message`, respectively?

Reply:
Made the required changes

Reply:
Made the required changes